### PR TITLE
ferdium-nightly: Add version 6.0.0-nightly.7

### LINKED ISSUE #494: [Request] Ferdium (nightly version)
### Information

**Name:** ferdium-nightly

**Description:** All-in-one messaging apps for various services

**Homepage:** https://github.com/ferdium/ferdium-app

**Download link(s):** https://github.com/ferdium/ferdium-app/releases/download/v6.0.0-nightly.7/Ferdium-Setup-6.0.0-nightly.7.exe

**Some indication of popularity/repute:** Currently about 50 stars on GitHub, but with this being a continuation/revival of [Ferdi](https://github.com/getferdi/ferdi) I expect it to continue rising in the future.

### PR DESCRIPTION
**Pull request by beerpiss**
Closes #494 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/ferdium-nightly.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "homepage": "https://github.com/ferdium/ferdium-app",
+    "license": "Apache-2.0",
+    "url": "https://github.com/ferdium/ferdium-app/releases/download/v6.0.0-nightly.7/Ferdium-6.0.0-nightly.7.exe#/dl.7z",
+    "hash": "23b1551d593558c1bccf3aff13c63e6dd4ada23e0fcdcba5adeecf360afddbfa",
+    "architecture": {
+        "64bit": {
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
+                ]
```
Comment (rashil2000) [inline bucket/ferdium-nightly.json:13]:
```suggestion
                "script": "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
```

Comment (rashil2000) [inline bucket/ferdium-nightly.json:13]:
```suggestion
                "script": "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
```

File: bucket/ferdium-nightly.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            }
+        },
+        "32bit": {
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\""
+                ]
```
Comment (rashil2000) [inline bucket/ferdium-nightly.json:20]:
```suggestion
                "script": "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\""
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Perfect manifest! Really nice work. Just some minor comments.

**Comment by beerpiss** [comment]
I've accepted all suggestions, thank you for taking some time to look through the manifest.

**Pull request merged**